tests/unit/customMatchers/toHaveBeenProgrammaticallyMoved/index.js

```diff
@@ -0,0 +1,37 @@
+import { matcherHint, printReceived } from 'jest-matcher-utils';
```

Comment:
Looking at the docs at https://jestjs.io/docs/en/expect#thisutils, it seems like these can be accessed through `this.utils`. Was that not working for you? Maybe it's only available after calling `expect.extend`?

Reply:
I'm pretty sure it would work too - it's basically doing the exact same thing, I've just been taking inspiration from the source mentioned below, and that was done using the direct import. Either way is fine, lemme know what you prefer style-wise.